### DISCUSSION TITLE
🔍💻 Look into Codespaces use for Github projects

### DISCUSSION THREAD

**Discussion by ghost** [OWASP/www-committee-project#41] [Ideas]

**Comment by bkimminich** [comment]
Juice Shop has a corresponding config in its main repo and also documentation at https://pwning.owasp-juice.shop/part3/contribution.html#developing-in-a-github-codespace for usage in Codespaces. I used this some time ago to work on my iPad.

I still prefer to work in my locally installed IntelliJ IDEA. And for tiny code changes "to go" I just used the GitHub website directly or a text editor (like Textastic) and Git client on my phone or iPad.

I've got no statistics if others really use Juice Shop in Codespaces, though.

**Comment by ghost** [comment]
Awesome. I was wondering if we could get codespaces credit through the OWASP GitHub org.

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/128622481/227787071-e7de81a5-9bc2-4ff6-a4e5-640c1f2d6b4c.png)
